components/atom/pinInput/test/index.test.js

```diff
@@ -581,18 +581,20 @@ describe('AtomPinInput', () => {
 
       // When
       const hook = setupReducerEnvironment(args)
-
       // Then
       const [store] = hook.result.current
+      console.log(store)
```

Comment:
Ok, this is a test but, we need this console.log()?

Reply:
XD, removed in: https://github.com/SUI-Components/sui-components/pull/1665